hikari/guilds.py

```diff
@@ -2642,7 +2642,9 @@ async def fetch_widget_channel(self) -> typing.Optional[channels_.GuildChannel]:
         if not self.widget_channel_id:
             return None
 
-        return await self.app.rest.fetch_channel(self.widget_channel_id)
+        widget_channel = await self.app.rest.fetch_channel(self.widget_channel_id)
+        assert widget_channel is None or isinstance(widget_channel, channels_.GuildChannel)
```

Comment:
No need to test this here (in any of the cases bellow either). `fetch_channel` will never actually return `None`

Reply:
This still applies to all of these channel methods, they shouldn't be asserting that it only be asserting that they're X not X or None since the fetch method will never return None